clients/cobol-lsp-vscode-extension/src/services/util/FSUtils.ts

```diff
@@ -151,7 +96,7 @@ function globSearch(
   pattern = pattern + suffix;
   const result = globSync(pattern, { cwd, dot: true });
   // TODO report the case with more then one copybook fit the pattern.
-  return result[0]
+  return result && result[0]
```

Comment:
```suggestion
  return result[0]
```
globSync seems to be typed to always return a valid array.

Reply:
It does, but there might be no element at index 0 always. can be an empty array. 
I think this might throwing runtime exception and it is not being noticed.

Reply:
If the array is empty, the result is `undefined`. Empty array is truthy anyway, right?

Reply:
trying to read undefined[0] is the problem.

Reply:
That should not happen, as the `globSync` is documented to always return valid array. If you are seeing undefined returned, is it possible done by a faulty mock in tests?

Reply:
it is removed : 28f73cfe6959a6939b38ecda4003811a73ba8e16